.travis.yml

```diff
@@ -1,12 +1,14 @@
 language: java
 jdk:
-- oraclejdk8
+  - oraclejdk8
+  - oraclejdk11
```

Comment:
For jdk11 we should us the openjdk11, since Oracle's JDK should be identical to the OpenJDK on Java 11. Also, oraclejdk11 is no longer maintained by Oracle since Java 12 is out.

Reply:
Done in 6bd41c1deb2a348d4f6dcee2f6e1f48c0376b6d3.